src/test/java/gov/nasa/jpl/mbee/mdk/mms/actions/GeneratePdfActionFromModelTest.java

```diff
@@ -0,0 +1,108 @@
+package gov.nasa.jpl.mbee.mdk.mms.actions;
+
+import com.nomagic.magicdraw.core.Application;
+import com.nomagic.magicdraw.core.Project;
+import com.nomagic.magicdraw.tests.MagicDrawTestRunner;
+import com.nomagic.magicdraw.uml.BaseElement;
+import com.nomagic.uml2.ext.magicdraw.classes.mdkernel.Element;
+import gov.nasa.jpl.mbee.mdk.api.MagicDrawHelper;
+import gov.nasa.jpl.mbee.mdk.options.MDKOptionsGroup;
+import org.apache.commons.io.FileUtils;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Create a PDF from MDK Document Model
+ * 1. create PDF from samples/MDK/DocGen.mdzip
+ * 2. Using the docbook XML file created by step 1, created PDF.
+ *
+ * @author Miyako Wilson
+ * @JIRA Cameo MDK MDK-64
+ */
+@RunWith(MagicDrawTestRunner.class)
+public class GeneratePdfActionFromModelTest {
+    /**
+     * Project on which test is performed.
+     */
+    private static Project project;
+    private static File docbookXslFo;
+    private static File testProjectFile;
+
+    /**
+     * Constructs this test.
+     */
+    public GeneratePdfActionFromModelTest() {
+    }
+
+    @BeforeClass
+    public static void setupProject() throws Exception {
+
+        String userdir = System.getProperty("user.dir");
+        File docGenProjectFile = new File(userdir + File.separator + "samples" + File.separator + "MDK" + File.separator + "DocGen.mdzip");
+        testProjectFile = File.createTempFile("DocGenCopied1", ".mdzip");
+        FileUtils.copyFile(docGenProjectFile, testProjectFile);
+
+        MDKOptionsGroup.getMDKOptions().setDefaultValues();
+        MDKOptionsGroup.getMDKOptions().setLogJson(true);
+        MagicDrawHelper.openProject(testProjectFile);
+        project = Application.getInstance().getProject();
+
+        //ClassLoader classLoader = GeneratePDFFromMDModelTest.class.getClassLoader();
+        docbookXslFo = new File(userdir + File.separator + "plugins" + File.separator + "gov.nasa.jpl.cae.magicdraw.mdk" + File.separator + "docbook-xsl-1.79.1" + File.separator + "fo" + File.separator + "mdk-default.xsl");
```

Comment:
See other comment about renaming.

Reply:
Removed docbook version dependency from the code.